packages/core/demo/demo-data/meter.ts

```diff
@@ -0,0 +1,22 @@
+import { colors } from "./colors";
+
+export const meterData = {
+	label: "Dataset 1",
+	fillColor: colors[2],
+	data: {
+		min: 0,
+		max: 1500,
+		peak: 1200,
+		value: 1000,
+		status: [
+			{ range: [0, 1000], status: "success" },
+			// { range: [500, 1000], status: "custom" },
+			{ range: [1000, 1200], status: "warning"},
+			{ range: [1200, 1500], status: "danger"}
+		],
+	}
```

Comment:
Some of these seem more option-y than data-y to me ... Is there any precedent for option-like data being used as part of the `data` object?

I guess really the question is, how much config should the user be expected to pass in when they call `setData` ... is it likely that the `status`, `min`, or `max` is going to be changing frequently? Since I'd expect a gauge like this to be receiving new data `value`s on a fairly regular basis

Reply:
I think `min`, `max`, `peak` and `status` should be in the chart options, and the chart data would basically just be a `number`.

Options could be under `meter`